res/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "sharedExtensions": [
+    {
+      "42": {
+        "trunks": [
+          {
+            "42": {
+              "stations": [
```

Comment:
While it may be better to deal with this when you start manipulating outbound dial, stations should not be conceptually a child of trunks. Trunks are a type of endpoint that participate in a shared extension; they are a separate concept from the stations that they serve.